src/solver/batchnorm/forward_inference_ck.cpp

```diff
@@ -327,7 +327,7 @@ bool BnCKFwdInference::IsApplicable(
         return false;
     if(!bn_problem.IsLayoutNHWC() && !bn_problem.IsLayoutNCHW())
         return false;
-    if(!ck_utility::is_ck_supported_hardware(context.GetStream()))
```

Comment:
`is_ck_supported_hardware` relates to the `Legacy CK`, while this solver uses `CK`.

Reply:
See https://github.com/ROCm/MIOpen/pull/3749#discussion_r2109613197